src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -902,6 +919,26 @@ public ConsumerDTO createConsumer(ConsumerDTO dto, String userName, String owner
         this.validator.validateCollectionElementsNotNull(dto::getInstalledProducts,
             dto::getGuestIds, dto::getCapabilities);
         Principal principal = this.principalProvider.get();
+        if (principal instanceof AnonymousCloudConsumerPrincipal anonymPrincipal) {
+            AnonymousCloudConsumer anonymousCloudConsumer = anonymPrincipal.getAnonymousCloudConsumer();
+            try {
+                ownerKey = cloudAdapter.validateRegisteredOwner(anonymousCloudConsumer.getCloudAccountId(),
+                    anonymousCloudConsumer.getCloudProviderShortName(),
+                    anonymousCloudConsumer.getCloudOfferingId());
+                if (!ownerCurator.existsByKey(ownerKey)) {
+                    throw new TooManyRequestsException(60);
```

Comment:
These 4 values should not be 'magic numbers'. Let's extract them as constants and explain what each duration supposed to be for

Reply:
done